tests/integration/test_data_cli.py

```diff
@@ -779,12 +785,192 @@ def test_search_update_fail(invoke, search_id, search_filter):
     assert result.exception
 
 
+@respx.mock
+@pytest.mark.asyncio
+@pytest.mark.parametrize("exists, overwrite",
+                         [(False, False), (True, False), (True, True),
+                          (False, True)])
+def test_asset_download_default(invoke, open_test_img, exists, overwrite):
+    # NOTE: this is a slightly edited version of test_download_asset from
+    # tests/integration/test_data_api
+    dl_url = f'{TEST_URL}/1?token=IAmAToken'
+
+    img_headers = {
+        'Content-Type': 'image/tiff',
+        'Content-Length': '527',
+        'Content-Disposition': 'attachment; filename="img.tif"'
+    }
+
+    async def _stream_img():
+        data = open_test_img.read()
+        v = memoryview(data)
+
+        chunksize = 100
+        for i in range(math.ceil(len(v) / (chunksize))):
+            yield v[i * chunksize:min((i + 1) * chunksize, len(v))]
+
+    # populate request parameter to avoid respx cloning, which throws
+    # an error caused by respx and not this code
+    # https://github.com/lundberg/respx/issues/130
+    mock_resp = httpx.Response(HTTPStatus.OK,
+                               stream=_stream_img(),
+                               headers=img_headers,
+                               request='donotcloneme')
+    respx.get(dl_url).return_value = mock_resp
+
+    dl_url = f'{TEST_URL}/1?token=IAmAToken'
+    basic_udm2_asset = {
+        "_links": {
+            "_self": "SELFURL",
+            "activate": "ACTIVATEURL",
+            "type": "https://api.planet.com/data/v1/asset-types/basic_udm2"
+        },
+        "_permissions": ["download"],
+        "md5_digest": None,
+        "status": 'active',
+        "location": dl_url,
+        "type": "basic_udm2"
+    }
+
+    runner = CliRunner()
+    with runner.isolated_filesystem() as folder:
+        if exists:
+            Path(folder, 'img.tif').write_text('i exist')
+
+        asset_download_command = [
+            'asset-download',
+            json.dumps(basic_udm2_asset),
+            '--directory',
+            Path(folder),
```

Comment:
these could be a little easier to read with options specified like this:

```suggestion
            f'--directory={Path(folder)}',
```
not necessary, just an fyi

Reply:
This is way better, thanks!